### ISSUE TITLE
[phys-services] compiler warnings (Trac #2076)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2068]
```text
/home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-
    services/I3VEMCalManipulator.cxx: In member function ‘void
    I3VEMCalManipulator::LoadDirectory(__cxx11::string)’:
/home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-services/I3VEMCalManipulator.cxx:175:39:
    warning: ‘int readdir_r(DIR *, dirent *, dirent **)’ is deprecated [-
    Wdeprecated-declarations]
while(readdir_r(dir, &file, &lfile) == 0)
^
In file included from /home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-services/I3VEMCalManipulator.cxx:4:0:
    /usr/include/dirent.h:183:12: note: declared here
extern int readdir_r (
        DIR *__restrict __dirp,
^~~~~~~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/2076
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:51", 
    "description": "{{{\n/home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-\n    services/I3VEMCalManipulator.cxx: In member function \u2018void\n    I3VEMCalManipulator::LoadDirectory(__cxx11::string)\u2019:\n/home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-services/I3VEMCalManipulator.cxx:175:39:\n    warning: \u2018int readdir_r(DIR *, dirent *, dirent **)\u2019 is deprecated [-\n    Wdeprecated-declarations]\nwhile(readdir_r(dir, &file, &lfile) == 0)\n^\nIn file included from /home/dschultz/Documents/combo/trunk/src/phys-services/private/phys-services/I3VEMCalManipulator.cxx:4:0:\n    /usr/include/dirent.h:183:12: note: declared here\nextern int readdir_r (\n        DIR *__restrict __dirp,\n^~~~~~~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067291068818", 
    "component": "combo core", 
    "summary": "[phys-services] compiler warnings", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2017-08-10T15:58:27", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170818T19:11:31: 
* david.schultz commented:

> In changeset 157685:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157685"
> readdir_r() is deprecated. fixes #2068
> ```


* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:51: 
* olivas changed milestone from "December 2017" to ""
* olivas commented:

> Milestone December 2017 deleted
